daemon/daemon.go

```diff
@@ -315,6 +315,8 @@ func (daemon *Daemon) restore() error {
 
 			if err := daemon.Register(container); err != nil {
 				logrus.Debugf("Failed to register container %s: %s", container.ID, err)
```

Comment:
I think the log level should be "ERROR" instead of "DEBUG".
Otherwise we have to ask the user to switch on the debug flag to report such kind of bugs, which is a bit inconvenient.